<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDaPePieChart</class>
 <widget class="QWidget" name="C_SyvDaPePieChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>7</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelShowUnit">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelShowValue">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="stw::opensyde_gui_elements::C_OgeChxTristateTransparentToggle" name="pc_CheckBoxUnit">
         <property name="minimumSize">
          <size>
           <width>66</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>66</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw::opensyde_gui_elements::C_OgeChxTristateTransparentToggle" name="pc_CheckBoxValue">
         <property name="minimumSize">
          <size>
           <width>66</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>66</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeChxTristateTransparentToggle</class>
   <extends>QCheckBox</extends>
   <header>C_OgeChxTristateTransparentToggle.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
